hack/e2e/run.sh

```diff
@@ -263,6 +284,8 @@ else
   loudecho "Not cleaning"
 fi
 
+OVERALL_TEST_PASSED="${TEST_PASSED}"
```

Comment:
Why is this now here?

Reply:
Otherwise we run into `OVERALL_TEST_PASSED: unbound variable` because the variable will not be set if `"${HELM_CT_TEST}" == true`.

Reply:
That doesn't answer the question, shouldn't it be right after line 238?